src/main/java/seedu/address/logic/commands/BudgetCalculationCommand.java

```diff
@@ -49,36 +52,39 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
 
         if (Integer.parseInt(totalBudget.toString()) <= 0) {
             throw new CommandException(MESSAGE_INVALID_TOTAL_BUDGET);
-        }
-        int i;
+        } else if (BudgetCalculationManager.getHaveBudgetsBeenCalculated(model)) {
+            return new CommandResult(String.format(MESSAGE_BUDGETS_ALREADY_CALCULATED));
+        } else {
+            int i;
 
-        int budgetPerPerson;
+            int budgetPerPerson;
 
-        CalculateTotalAttendees totalAttendees = new CalculateTotalAttendees(listOfClubs);
+            CalculateTotalAttendees totalAttendees = new CalculateTotalAttendees(listOfClubs);
 
-        budgetPerPerson = Integer.parseInt(totalBudget.toString()) / totalAttendees.arithmeticTotalAttendees();
+            budgetPerPerson = Integer.parseInt(totalBudget.toString()) / totalAttendees.arithmeticTotalAttendees();
```

Comment:
You can rename your method name from "arithmeticTotalAttendees()" to "calculateTotalAttendees()" and your class name from "CalculateTotalAttendees" to "TotalAttendees" so that your method sounds more action like

Reply:
Thank you for pointing this out! I have made the suggested changes